app/models/ai_persona.rb

```diff
@@ -171,6 +232,25 @@ def class_instance
     end
   end
 
+  def create_user!
+    raise "User already exists" if user_id && User.exists?(user_id)
```

Comment:
```suggestion
    raise "User already exists" if User.exists?(user_id)
```

`User.exists?(nil) # false`

Reply:
looks correct as a change but feels a bit too magical to me, I like the extra deliberate code in this specific case.